app/models/concerns/retrieve_media.rb

```diff
@@ -8,12 +8,6 @@ module RetrieveMedia
   end
 
   def retrieve_media
-    if anime.present?
-      anime
-    elsif manga.present?
-      manga
-    elsif drama.present?
-      drama
-    end
+    anime.presence || manga.presence || drama.presence || nil
```

Comment:
`|| nil` isn't needed tbh

Reply:
wasn't sure so just put it there to be safe. Will remove though.